src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/AnswerService.cs

```diff
@@ -0,0 +1,16 @@
+using System.Globalization;
+using System.Text.RegularExpressions;
+using DevAdventCalendarCompetition.Services.Interfaces;
+
+namespace DevAdventCalendarCompetition.Services
+{
+    public class AnswerService : IAnswerService
+    {
+        public string ParseTestAnswer(string answer)
+        {
+            return string.IsNullOrWhiteSpace(answer) ?
+                string.Empty :
+                Regex.Replace(answer.ToUpper(CultureInfo.CurrentCulture).Trim(), "\\s{2,}", " ");
```

Comment:
don't use that! it will work different on different servers. Always use InvariantCulture

Reply:
you can probably replace "\\s{2,}" with @"\s{2,}"